gbmodel/model_sqlalchemy.py

```diff
@@ -182,6 +187,33 @@ def get_team_members(self, tid):
             return None
         return mems
 
+    def get_students_in_session(self, session_id):
+        """
+        Gets a list of students in the given session, ordered by team (in ascending order)
+        Input: session_id
+        Output: the list of students
+        """
+        # https://stackoverflow.com/questions/4186062/sqlalchemy-order-by-descending
+        # https://docs.sqlalchemy.org/en/13/orm/query.html
+        try:
+            results = students.query.filter(
+                          students.session_id == session_id).order_by(students.tid.asc()).all()
```

Comment:
If I may ask, what kind of indentation strategy do we want here? Does what I did here work?

Reply:
I think that's fine